infrastructure/logging-bucket.yaml

```diff
@@ -0,0 +1,32 @@
+# Define a Log Bucket in GCP to store application logs
+apiVersion: logging.cnrm.cloud.google.com/v1beta1
+kind: LoggingLogBucket
+metadata:
+  name: hopic-application-logs-pht-01hp04dtnkf
+  namespace: cnrm-system
+spec:
+  projectRef:
+    external: "projects/pht-01hp04dtnkf"
+  location: "northamerica-northeast1"
+  description: "Hopic Application Logs Storage"
+  # Whether the log bucket is protected from deletion
+  locked: true
+  # Number of days to retain logs in the bucket
+  retentionDays: 90
+---
+# Define a Log Sink for routing logs to the defined Log Bucket
+apiVersion: logging.cnrm.cloud.google.com/v1beta1
+kind: LoggingLogSink
+metadata:
+  namespace: cnrm-system
+  name: hopic-application-logs-sink
+spec:
+  projectRef:
+    external: "pht-01hp04dtnkf"
```

Comment:
Could we modify this to `projects/<project-id>`, similar to what to you have for `LoggingLogBucket` resource? Just to have a consistent way of referencing external resources.

Reply:
Initially I tried with using external: "projects/pht-01hp04dtnkf" but the config connector failed to deploy. Here are the logs

 Warning  UpdateFailed  22s (x11 over 10m)  logginglogsink-controller  Update call failed: error fetching live state: error importing resource: unable to parse logging sink id "proj │
│ ects/pht-01hp04dtnkf###hopic-application-logs-sink"

Reply:
That's weird! I was under the impression that we're creating this resource. Why is it importing it lol.

Reply:
You can disregard my suggestion for this. Looks like it's upstream issue: https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/614. 

Doesn't seem like it's gonna be fixed anytime soon lol.